python-sdk/src/astro/databases/snowflake.py

```diff
@@ -766,6 +818,68 @@ def _build_merge_sql(
         }
         return statement, params
 
+    def append_table(
+        self,
+        source_table: BaseTable,
+        target_table: BaseTable,
+        source_to_target_columns_map: dict[str, str],
+    ) -> None:
+        """
+        Append the source table rows into a destination table.
+        The argument `if_conflicts` allows the user to define how to handle conflicts.
```

Comment:
Where is the `if_conflicts` argument?

Reply:
This metho was copied from base.py. Removing it both. Thanks for pointing out.